app/exporters/formatters/utaac_decision_indexable_formatter.rb

```diff
@@ -9,13 +9,13 @@ def type
   def extra_attributes
     {
       indexable_content: "#{entity.hidden_indexable_content}\n#{entity.body}".strip,
-      tribunal_decision_category: entity.tribunal_decision_category,
-      tribunal_decision_category_name: expand_value(:tribunal_decision_category).first,
+      tribunal_decision_categories: entity.tribunal_decision_categories,
+      tribunal_decision_categories_name: expand_value(:tribunal_decision_categories),
```

Comment:
The renaming does make sense.

However there are knock on effects:
* we'd have to rename or add a new field definition in rummager: https://github.com/alphagov/rummager/blob/6fb0e127650384a1f943d73a69f5e37359e2b543/config/schema/field_definitions.json#L435
* we'd have to commit changes for other tribunals following this pattern: https://github.com/alphagov/specialist-publisher/search?utf8=%E2%9C%93&q=tribunal_decision_categories_name

I think given the effort/return tradeoff it's ok to leave it as `tribunal_decision_categories_name`. What do you think?